### PR TITLE
Fixed audio distortion using speaker phone.

### PR DESCRIPTION
**Pull request by kbagchiGWC**

### CODE REVIEW

**Comment by tmconnors** [comment]
@kbagchiGWC Use block comments. Otherwise +1

**Comment by dreamerns** [comment]
Why is it being set at the end of the call? It should be set when call gets established.

**Comment by tmconnors** [comment]
Good catch @dreamerns, what about setting this in `toggleSpeakerPhoneFabClickListener` @kbagchiGWC?

**Comment by kbagchiGWC** [comment]
I will need to debug this further. Theoretically without setting it in `toggleSpeakerPhoneFabClickListener` it should not even work but it did.

**Comment by kbagchiGWC** [comment]
@tmconnors @dreamerns : updated the PR.

**Comment by dreamerns** [comment]
👍

File: app/src/main/java/client/twilio/com/quickstart/ClientActivity.java [hunk 1]
```diff
@@ -666,4 +672,26 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             }
         }
     }
+
+    private void setAudioFocus(boolean setFocus) {
+        if (audioManager != null) {
```
Comment (tmconnors) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: Combine conditionals.

File: app/src/main/java/client/twilio/com/quickstart/ClientActivity.java [hunk 2]
```diff
@@ -129,6 +132,11 @@ protected void onCreate(Bundle savedInstanceState) {
         clientProfile = new ClientProfile("jenny", true, true);
 
         /*
+         * Needed for setting/abandoning audio focus during call
+         */
+        audioManager = (AudioManager)getSystemService(Context.AUDIO_SERVICE);
```
Comment (tmconnors) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: Formatting.

Comment (kbagchiGWC) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: What formatting?

File: app/src/main/java/client/twilio/com/quickstart/ClientActivity.java (hunk 1 above)
Comment (kbagchiGWC) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: Combine? By repeating `audioManager != null` check?

File: app/src/main/java/client/twilio/com/quickstart/ClientActivity.java (hunk 2 above)
Comment (tmconnors) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: Spacing `(AudioManager) getSystemService`

File: app/src/main/java/client/twilio/com/quickstart/ClientActivity.java (hunk 1 above)
Comment (tmconnors) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: Ignore, misread.

File: app/src/main/java/client/twilio/com/quickstart/ClientActivity.java (hunk 2 above)
Comment (kbagchiGWC) [inline app/src/main/java/client/twilio/com/quickstart/ClientActivity.java]: Android Studio did not suggest the spacing. Refer to video-QS https://github.com/twilio/video-quickstart-android/commit/c6a0e1da911d9f1da48b3ca4a232170a3b4be5b3#diff-8fc9af9db30a338466cb0bc45ed92dd6R141, has the same formatting as this.

**Comment by tmconnors** [comment]
+1

**Pull request merged**